arviz/plots/forestplot.py

```diff
@@ -289,11 +289,18 @@ def make_plotters(self):
 
     def labels_and_ticks(self):
         """Collect labels and ticks from plotters."""
-        labels, idxs = [], []
-        for plotter in self.plotters.values():
-            sub_labels, sub_idxs, _, _ = plotter.labels_ticks_and_vals()
-            labels.append(sub_labels)
-            idxs.append(sub_idxs)
+        val = self.plotters.values()
+
+        @conditional_jit(forceobj=True)
+        def label_idxs():
+            labels, idxs = [], []
+            for plotter in val:
+                sub_labels, sub_idxs, _, _ = plotter.labels_ticks_and_vals()
+                labels.append(sub_labels)
+                idxs.append(sub_idxs)
+            return labels, idxs
```

Comment:
Would `return np.concatenate(labels), np.concatenate(idxs)` here (where is compiled by numba) instead of 4 lines later (where it is not) make any difference?